### ISSUE TITLE
Remove Unused Ambers

### ISSUE THREAD

**Issue by sk2048** [Darkosto/SkyFactory4Issues#752]
## Bug Report

The following amber blocks should be removed (as per SkyFactory 4 - Metals spreadsheet)

They are mostly useless or only meltable in nuclearcraft melter to produce same molten metal as sum of components.

- [ ] Bone
- [ ] Clay
- [ ] Diamond
- [ ] Emerald
- [ ] Gravel
- [ ] Lapis
- [ ] Quartz

**Comment by Crackerhead22** [comment]
Were these not left in for decoration?

**Comment by Masciguy** [comment]
I use the clay amber recipe. It helps eat up the acorns and can be broken into clay balls in the standard crafting.

**Comment by sk2048** [comment]
If they were left in for decoration, I missed that. If that is the case, then they should probably also be functional to stop people creating issues. That, or have a tooltip stating they are for decoration and offer no benefit.

**Comment by Vahtera** [comment]
Lapis Acorns to Lapis Block recipe is not visible anywhere, should be added to JEI if it is intended to use like this. I was confused for a long time about how to acquire lapis, because this recipe is not visible.

**Comment by Crackerhead22** [comment]
> Lapis Acorns to Lapis Block recipe is not visible anywhere, should be added to JEI if it is intended to use like this. I was confused for a long time about how to acquire lapis, because this recipe is not visible.

That's odd, I see it it in JEI

**Comment by Almighty404** [comment]
Also, Bone Amber is a source of bones. So it is useful.

**Comment by Vahtera** [comment]
> > Lapis Acorns to Lapis Block recipe is not visible anywhere, should be added to JEI if it is intended to use like this. I was confused for a long time about how to acquire lapis, because this recipe is not visible.
> 
> That's odd, I see it it in JEI

I see it if I look at "recipes for lapis block", but I DO NOT see it if I look at "what can be made with Lapis Acorns" (if you know what I'm trying to say)

**Comment by CookedPages** [comment]
I like to use all the ambers for decoration. It's a really interesting style of building that I appreciate, as this skyblock doesn't actually have a lot of unique/varied decorative options outside of the Chinese workshop. Oh I guess Twighlight blocks are interesting too, but a pain to get, even with op weapons and flight.

**Comment by iamc24** [comment]
They are all usable for something except for gravel amber which has no purpose besides decoration. Gravel amber needs a use!

**Comment by PBGgamer** [comment]
> Also, Bone Amber is a source of bones. So it is useful.

Yeah, I was hoping to make bone crooks with this. Reinstalled my versions because I thought it was a script error lol.
How else do you get bones? I guess I'll settle for skeletons.
